package.json

```diff
@@ -25,6 +26,7 @@
     "test": "jest -i",
     "test:coverage": "dotenv -e .env.example -- jest -i --coverage",
     "start": "node src/server.js",
+    "generate": "npm-run-all generate:*",
     "generate:proxy-models": "npm-run-all generate:proxy:* generate:api:* generate:messages:*",
```

Comment:
We can avoid redundant generate calls.
```suggestion
    "generate": "npm-run-all generate:proxy:* generate:api:* generate:messages:*",
```

Reply:
I think this way we're going to miss test-certs (which we can actually generate specifically only on tests)

Reply:
You are right, I've missed it 👍

Reply:
But still, I think we should not generate test-certs when deploying, so it makes sense to generate it separately. Maybe on `pretest`?

Reply:
On `pretest` task new certificates will be generated every time test are executed locally.

Reply:
In the end, I decide not to. I just executed such script before tests